torch/_dynamo/variables/user_defined.py

```diff
@@ -489,6 +489,18 @@ def call_function(
         elif issubclass(self.value, enum.Enum) and len(args) == 1 and not kwargs:
             options = {"mutable_local": MutableLocal()}
             return variables.EnumVariable.create(self.value, args[0], options)
+        elif (
+            not self.is_standard_new()
+            and SideEffects.cls_supports_mutation_side_effects(self.value)
```

Comment:
I want to relax this constraint in a future PR. I don't understand why we need this yet.

Reply:
So why does this constraint exist?

Reply:
I could not get a chance to follow up. I remember seeing many tests failing. I can follow up on this now.